v8/postcast/postcast.py

```diff
@@ -178,14 +178,20 @@ def enclosure(self, post=None, lang=None):
 
     def audio_url(self, lang=None, post=None):
         config = self.site.config
-        return urljoin(config['BASE_URL'], self.audio_path(lang=lang, post=post, is_link=True))
+        baseurl = config.get('POSTCAST_BASE_URL') or config['BASE_URL']
```

Comment:
You can simplify this with: `config.get('POSTCAST_BASE_URL', config['BASE_URL'])`

You can also do the same below.

Reply:
Thanks for the quick response!

> You can simplify this with: `config.get('POSTCAST_BASE_URL', config['BASE_URL'])`

It's not strictly the same, if `POSTCAST_BASE_URL` was set to an empty string this would still default to the `BASE_URL`, but it's fine with me either way.

> You can also do the same below.

In the code below `POSTCAST_ENCLOSURE_FOLDER` is not simply an alternative data source, if it's present it changes the behavior of how the path is constructed, I don't see how that can be done with a default for `config.get`.